engine/servermaster/http.go

```diff
@@ -86,3 +98,68 @@ func shouldForwardJobAPI(r *http.Request) bool {
 	}
 	return true
 }
+
+// JobHTTPClient is the http client for job operations, like 'get job detail'
+type JobHTTPClient interface {
+	// GetJobDetail sends http request to the specific jobmaster to get job detail
+	// path format for get job detail openapi is: 'api/v1/jobs/${JobID}/status'
+	// If no error happens and the response status code is 2XX, return the response body as the job detail
+	GetJobDetail(ctx context.Context, jobMasterAddr string, jobID string) ([]byte, error)
+	// Close releases the resources
+	Close()
+}
+
+// jobHTTPClientImpl is the http client for getting job detail
+type jobHTTPClientImpl struct {
+	cli        *httputil.Client
+	pathFormat string
+}
+
+// NewJobHTTPClient news a jobHTTPClientImpl
+func NewJobHTTPClient(cli *httputil.Client, pathFormat string) *jobHTTPClientImpl {
```

Comment:
What is the significance of `pathFormat`? Can it be defined as a constant?

Reply:
Ok.

Reply:
Fix it.